docs/src/rsh/errors/index.md

```diff
@@ -2116,6 +2116,18 @@ Maybe(UInt).Some(5).match({
 This error indicates that you have inspected the details about a publication, such as via `{!rsh} didPublish()`, before there has been a publication.
 This is impossible, so the expression must be moved after the first publication.
 
+For example, the code below erroneously puts `{!rsh} didPublish()` before `{!rsh} publish`:
+
+``` reach
+load: /hs/t/n/Err_NotAfterFirst.rsh
+```
+
+This error can be corrected by placing `{!rsh} publish` before `{!rsh} didPublish()`:
+
+```reach
+load: /hs/t/n/j610.rsh
```

Comment:
This program is also incorrect though, so I don't think you should use this example. (All of the programs in `n` don't work and the ones in `y` do)